jessie.json

```diff
@@ -20,8 +20,8 @@
             "headless": "true",
             "disk_size": "{{ user `disk_size`}}",
             "qemuargs": [[ "-m", "1024M" ],[ "-smp", "4" ]],
-            "iso_url": "http://cdimage.debian.org/cdimage/jessie_di_rc1/amd64/iso-cd/debian-jessie-DI-rc1-amd64-netinst.iso",
-            "iso_checksum": "2ab639aa971279f2cfbe45ea01a51529",
+            "iso_url": "http://debian.anexia.at/debian-cd/current/amd64/iso-cd/debian-8.2.0-amd64-netinst.iso",
```

Comment:
This should probably still be cdimage.debian.org which will select the closest mirror to the user